python/ray/train/_internal/backend_executor.py

```diff
@@ -153,6 +155,19 @@ def start(
 
             if self._num_gpus_per_worker > 0 and share_cuda_visible_devices_enabled:
                 self._share_cuda_visible_devices()
+            elif self._additional_resources_per_worker:
+                for (
+                    accelerator,
+                    accelerator_config,
+                ) in SUPPORTED_ACCELERATOR_DEVICES_TO_CONFIG.items():
+                    enable_sharing_env = accelerator_config[0]
+                    accelerator_runtime_env_var = accelerator_config[1]
+                    if self._share_accelerator_devices_enabled(
+                        accelerator, enable_sharing_env
+                    ):
+                        self._share_resource_ids(
+                            accelerator, accelerator_runtime_env_var
+                        )
```

Comment:
nit: What do you think about naming everything consistently as "resource" rather than "accelerator"?